src/view/utils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import ViewPosition from './position';
+import ViewTreeWalker from './treewalker';
+
+/**
+ * Contains utility functions for working on view.
+ *
+ * @module engine/view/utils
```

Comment:
Let's rename this to `walker-utils` since the utils that come to our mind are Walker related. We also considered putting this to the walker's module but we don't want to make it too big (hence, a new module).